CMakeLists.txt

```diff
@@ -134,6 +134,16 @@ include(cmake/SetupCMakeBasics.cmake)
 include(cmake/SetupCompilerFlags.cmake)
 include(cmake/SetupUmpireThirdParty.cmake)
 
+if (WIN32)
+  if ((NOT UMPIRE_ENABLE_FILESYSTEM) OR (CMAKE_CXX_STANDARD LESS 17))
+    message(FATAL_ERROR "\
+    Windows builds require the FileSystem to be enabled \
+    and the C/C++ standard to be set to 17 or higher. \
+    Please re-configure with UMPIRE_ENABLE_FILESYSTEM=ON, \
+    with BLT_CXX_STD=c++17 and CMAKE_CXX_STANDARD=17 or greater.")
```

Comment:
```suggestion
    with BLT_CXX_STD=c++17 or greater.")
```

Reply:
Check is good but this message is clearer, since use doesn't have to set both variables.

Reply:
si senor